src/main/java/com/garannodou/role/domain/User.java

```diff
@@ -0,0 +1,38 @@
+package com.garannodou.role.domain;
+
+import java.net.URI;
+import java.util.UUID;
+
+import static java.util.UUID.randomUUID;
+
+public class User {
+    private UUID id;
+
+    private String userName; //TODO: Encrypt field
+
+    private String nickName; //TODO: Ask to user, by default userName
+
+    private String email; //TODO: Encrypt field
+
+    private URI imageUrl;
+
+    public User(String userName, String nickName, String email, URI imageUrl) {
```

Comment:
Add unit test for validations.

Reply:
They are already added, along with lombok getters annotations. Additionally I have added the GoogleUserAuthenticator test class.